docs/pages/concepts/architecture/architecture.md

```diff
@@ -54,6 +55,15 @@ Meshery Operator is the multi-service mesh operator (a Kubernetes custom control
 [![Meshery Operator and MeshSync]({{ site.baseurl }}/assets/img/architecture/meshery-operator-and-meshsync.svg
 )]({{ site.baseurl }}/assets/img/architecture/meshery-operator-and-meshsync.svg)
 
+See the [**Operator**]({{ site.baseurl }}/architecture/operator) section for more information on the function of an operator and [**MeshSync**]({{ site.baseurl }}/architecture/meshsync) section for more information on the function of meshsync.
+
+## Meshery Database
+Meshery Database is responsible to persist and manage the state of meshery and its components.
+
+[![Meshery Database]({{ site.baseurl }}/assets/img/architecture/meshery-database.svg)]({{ site.baseurl }}/assets/img/architecture/meshery-database.svg)
+
+See the [**Database**]({{ site.baseurl }}/architecture/database) section for more information on the function of the database.
+
 ### **Network Ports**

```

Comment:
@kumarabd please review existing and capture new network ports for database (internal only), operator, broker.

Reply:
@leecalcote It is already been captured for broker. And we don't need any for the database.

Reply:
Ok. Rather, the operator reuses the existing https port existing between K8s and Meshery.